### PR TITLE
feat: add SubsidyRequestManagementTable

### PR DESCRIPTION
**Pull request by adamstankiewicz**
* Adds new `EnrollmentRequestManagementTable` component that will be used between subscriptions and codes

Ticket: https://openedx.atlassian.net/browse/ENT-5224

#### "Manage Requests" table

image

#### Course details popover

image

# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [x] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/subscriptions/tests/SubscriptionManagementPage.test.jsx [hunk 1]
```diff
@@ -1,12 +1,9 @@
-import {
```
Comment (adamstankiewicz) [inline src/components/subscriptions/tests/SubscriptionManagementPage.test.jsx:1]: The changes in this file resolve prop type and `act` warnings in the tests.

File: .env.development [hunk 2]
```diff
@@ -22,20 +23,21 @@ FAVICON_URL='https://edx-cdn.org/v3/default/favicon.ico'
...
-FEATURE_REPORTING_CONFIGURATIONS=true
-FEATURE_ANALYTICS=true
-FEATURE_SUPPORT=true
-FEATURE_SAML_CONFIGURATION=true
-FEATURE_EXTERNAL_LMS_CONFIGURATION=true
-FEATURE_BULK_ENROLLMENT=true
-FEATURE_SETTINGS_PAGE=true
-FEATURE_SETTINGS_UNIVERSAL_LINK=true
+PORT='1991'
+FEATURE_CODE_MANAGEMENT='true'
```
Comment (adamstankiewicz) [inline .env.development:27]: Changed the values here to explicitly be strings, as that is how they're parsed (not an actual boolean).

**Review by macdiesel — APPROVED** [review]
Seems reasonable.

**Comment by macdiesel** [comment]
That said you'll likely want a review from someone who knows more react than I.

**Review by manny-m — APPROVED** [review]
Cool! LGTM 👍

**Pull request merged**